tornado/test/websocket_test.py

```diff
@@ -92,6 +92,12 @@ def open(self, arg):
         self.write_message(arg)
 
 
+class HandshakeResponseHeadersHandler(TestWebSocketHandler):
+
+    def prepare(self):
+        self._headers['Set-Cookie'] = 'hello=1'
```

Comment:
When I said to use `prepare`, I meant to call the usual `set_header`/`add_header` methods from `prepare()`.